core/src/main/java/com/gcinterceptor/core/GarbageCollectorControlInterceptor.java

```diff
@@ -19,6 +20,10 @@
 	private Sampler sampler;
 	private Executor executor;
 	
+	static {
+		DEBUG_GCI = Boolean.parseBoolean(System.getenv("debug_gci"));
```

Comment:
No need to have this is a static block. The [Boolean.parseBoolean](https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#parseBoolean(java.lang.String)) method does not throw any error.

Please move up before merge.